CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ numbering uses [semantic versioning](http://semver.org).
 
 NOTE: The change log until version `v0.2.4` is auto-generated.
 
+## [v1.8.4](https://github.com/Xtigyro/puppetserver-helm-chart/tree/v1.8.4) (2020-05-01)
```

Comment:
should this be updated to puppetlabs?

Reply:
Ah, yes - missed it in this one - apologies! Will fix it now.

Reply:
Done.